### DISCUSSION TITLE
Variável constante

### DISCUSSION THREAD

**Discussion by lumafelic** [DesignLiquido/delegua#324] [Ideias]
Percebi que não há diferenciação nas variáveis como para valores que podem ser mudados ou não. 

Assim como em javaScript há var, let e const

Ou ser apenas nas diferenças de minúsculo ou maiúsculo na nomenclatura?

**Comment by leonelsanchesdasilva** [comment]
# Sobre `var`

`var` é algo que surgiu nos primórdios de JavaScript e virou um grande problema depois. Ser capaz de declarar coisas diretamente no escopo global vira um problema enorme de colisão de variáveis em código mais sofisticados e complexos. A não ser que tenhamos um ótimo motivo pra implementar o `var` de JavaScript, prefiro evitar a implementação por enquanto.

# Sobre `const`

`const` eu já acho interessante de implementar. Imutabilidade de variável pode ser algo muito positivo para a linguagem. 

# Sobre `let`

`let` é o nosso `var` de Delégua.

**Reply by samuelrvg** [reply]
Ter `const` é uma boa, já podemos até abrir uma ISSUE para essa implementação.

**Reply by ItaloCobains** [reply]
Creio que o `const` vai ser bem interessante, porém seguindo o intuito principal da linguagem que é o português eu usaria `constante` no lugar de `const` porem escrever uma palavra tão grande repetidas vezes seria desgastante então acho que algo mais abreviado como `fixo` seria melhor. Conversei com o @leonelsanchesdasilva e ele sugeriu manter `const` e `constante`, usando `fixo` como sinônimo.

**Reply by samuelrvg** [reply]
Se vamos usar `const` e `constante`, logo `var` e `variavel` também é uma boa.

**Comment by leonelsanchesdasilva** [comment]
Suporte implementado na versão 0.17 em diante.
